src/charm.py

```diff
@@ -1459,15 +1445,33 @@ def _was_restore_successful(self, container: Container, service: ServiceInfo) ->
             logger.debug("Restore check early exit: Patroni has not started yet")
             return False
 
+        restoring_backup = self.app_peer_data.get("restoring-backup")
+        restore_timeline = self.app_peer_data.get("restore-timeline")
+        restore_to_time = self.app_peer_data.get("restore-to-time")
+        # Sometimes this check fails to connect to the PostgreSQL. Not happened on VM, so it's a k8s-specific try-catch
+        # try:
+        #     current_timeline = self.postgresql.get_current_timeline()
+        # except PostgreSQLGetCurrentTimelineError:
+        #     logger.debug("Restore check early exit: can't get current wal timeline")
+        current_timeline = self.postgresql.get_current_timeline()
+
         # Remove the restoring backup flag and the restore stanza name.
         self.app_peer_data.update({
             "restoring-backup": "",
             "restore-stanza": "",
             "restore-to-time": "",
+            "restore-timeline": "",
         })
         self.update_config()
         self.restore_patroni_on_failure_condition()
-        logger.info("Restore succeeded")
+
+        logger.info(
+            "Restored"
+            f"{' to ' + restore_to_time if restore_to_time else ''}"
```

Comment:
It would be a little bit weird to show `Restored to from timeline...`. Should the else clause return `latest` instead of an empty string?

Reply:
If user passed `latest` in the `restore_to_time` parameter - it will be in this variable (like `Restored to latest from backup ...` or `Restored to latest from timeline ...`). If it's an ordinary restore without PITR, it will entirely skip this line and just output `Restored from backup ...`.

Reply:
Got it! Thanks!